src/lid.c

```diff
@@ -89,61 +89,25 @@ enum LidLayerTypes {
     STOR,                    // storage layer
```

Comment:
GENERAL COMMENT: It looks like your compiler is setup to auto-format files.  We haven't adopted a format standard at this point in the project.  If possible, I think it would be good if you could turn that off for now.  This will keep the diff's cleaner and simple.  Also I suggest resetting this file

Reply:
I removed the auto-format files in Visual Studio. Hopefully it is easier to see the differences now.